test/jdk/javax/swing/JMenuItem/bug4839464.java

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright (c) 2004, 2023, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+    @test
+    @bug 4839464
+    @summary Shortcoming in the way JMenuItem handles 'propertyChange()' events.
+    @library ../../regtesthelpers
+    @build Util JRobot
```

Comment:
I see Util is added but I don't see any Util methods used. Is this needed?

Also, did you try running this test in mach5 after moving it to open? I think you get an error with this ```regtesthelpers``` path since you removed the individual directories for each of these tests and placed them directly into the ```JMenuItem``` dir.

Reply:
@kumarabhi006 Yes, it should be `@library ../regtesthelpers`

Reply:
@DamonGuy @honkar-jdk Thanks for pointing out. I missed to run in open. Updated the tests and these tags are removed as they are of no longer use.